testing/smoketest/variables.tf

```diff
@@ -24,7 +30,7 @@ variable "ess_region" {
 variable "ess_deployment_template" {
   type        = string
   description = "Elastic Cloud deployment template"
-  default     = "gcp-compute-optimized-v3"
+  default     = "gcp-vector-search-optimized"
```

Comment:
What advantages has this instance type in comparison to the other available instance types?

Reply:
:+1: